src/Normalizer/InstantArticleContentEntityNormalizer.php

```diff
@@ -144,6 +144,11 @@ public function normalize($data, $format = NULL, array $context = []) {
     // If we're given an entity_view_display object as context, use that as a
     // mapping to guide the normalization.
     if ($display = $this->entityViewDisplay($data, $context)) {
+      // Declare a dependency on the view mode configuration if we are rendering
+      // in the context of a views REST export.
+      if (isset($context['views_style_plugin'])) {
+        $context['views_style_plugin']->displayHandler->display['cache_metadata']['tags'] = array_merge($context['views_style_plugin']->displayHandler->display['cache_metadata']['tags'], $display->getCacheTags());
```

Comment:
I think you should possibly use [Cache::mergeTags()](https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Cache%21Cache.php/function/Cache%3A%3AmergeTags/8.2.x) instead of array_merge()? I would normally recommend using [CacheableMetadata::merge()](https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Cache%21CacheableMetadata.php/function/CacheableMetadata%3A%3Amerge/8.2.x) but I think the Views style plugin's display handler probably doesn't implement `\Drupal\Core\Cache\CacheableMetadata`.